builder/oracle/oci/config.go

```diff
@@ -48,6 +48,9 @@ type Config struct {
 	// Instance
 	InstanceName string `mapstructure:"instance_name"`
 
+	// MetaData is optional but will be constructed to include UserData or UserDataFile under the "user_data" key
```

Comment:
Perhaps:
```go
// Metadata optionally contains custom metadata key/value pairs that you
// provide. A metadata service, available at http://169.254.169.254, runs on the
// launched instance. You can use the service to:
// Provide information to Cloud-Init to be used for various system initialization
// tasks (via the "user_data" key).
// Get information about the instance, including the custom metadata that you
// provide when you launch the instance.
```

Reply:
I tried to make it both more descriptive and not too detailed. I tried to keep the info about what takes precedence, as I agree with @rickard-von-essen  that this is the best way to go forward and important to convey. It's a mix of both our initial suggestions.